### PR TITLE
Events table antd

### PR DESCRIPTION
**Pull request by timgl**
## Changes

- events table to antd (including date break rows)
- Use antd tabs for properties/events when clicking on an event
- PropertiesTable to antd

## Checklist
- [ ] All querysets/queries filter by Team (if applicable)
- [ ] Backend tests (if applicable)
- [ ] Cypress E2E tests (if applicable)

### CODE REVIEW

File: frontend/src/scenes/events/EventsTable.js [hunk 1]
```diff
@@ -2,94 +2,160 @@ import React from 'react'
...
+                            You don't have any items here. If you haven't integrated PostHog yet,{' '}
+                            <Link to="/setup">click here to set PostHog up on your app</Link>
+                        </span>
+                    ),
+                }}
+                pagination={{ pageSize: 99999, hideOnSinglePage: true }}
+                rowKey={row => (row.event ? row.event.id : row.date_break)}
+                rowClassName={row => {
+                    if (row.event) return 'event-row'
+                    if (row.date_break) return 'event-day-separator'
```
Comment (EDsCODE) [inline frontend/src/scenes/events/EventsTable.js:143]: I have been proven wrong!

File: frontend/src/scenes/events/EventRow.js [hunk 2]
```diff
@@ -1,74 +0,0 @@
...
-    index,
-}) {
-    let params = ['$current_url', '$lib']
-    return (
-        <tr
-            className={'cursor-pointer event-row ' + (highlightEvents[event.id] && 'event-row-new')}
-            onClick={() => setSelectedEvent(selectedEvent !== event.id ? event.id : false)}
-            data-attr={'event-row-' + index}
-        >
-            <td data-attr={'event-name-' + index}>
```
Comment (EDsCODE) [inline frontend/src/scenes/events/EventRow.js:32]: The cypress test just needs this to be brought back

**Review by EDsCODE — CHANGES_REQUESTED** [review]
A few small changes regarding the event table itself. This might be PR creep but I'd consider wrapping in a live action table refactor since it's very similar to the events table and it's looking like this rn: 
Screen Shot 2020-05-29 at 11 35 16 AM

Also, once these two tables are done the antd coverage is pretty much complete (component wise). There's still a decent amount of bootstrap class styling being used but most if not all of the components will be on the antd standard

**Comment by timgl** [comment]
@EDsCODE Done! That live actions page should also be a lot quicker now

**Comment by mariusandra** [comment]
Looks pretty cool! 

Some comments below, all UX nits and can be solved outside this PR:

- For added UX points, I'd add some "filter" button next to the properties, so that I could get all events with Browser=Chrome easily.

image

- The top in the "Person" page looks rather cramped. It's somehow not immediately clear if you click "test@email.com" in the table that the events are now scoped to the person from the slight header change.

image

**Review by EDsCODE — APPROVED** [review]

**Comment by EDsCODE** [comment]
Agree with @mariusandra! I added a cursor pointer but otherwise looks good and the adjustments can be made as a followup

**Comment by timgl** [comment]
Updated the spacing and header size, left the clicking on property table for now as that's slightly more involved. #902 created for that.

**Pull request merged**